### DISCUSSION TITLE
Timeline plugin not working with JS?

### DISCUSSION THREAD

**Discussion by ksaugmann** [katspaugh/wavesurfer.js#3279] [Q&A] [answered]
I'm trying to use the timeline plugin and reference it like this:

<script src="https://unpkg.com/wavesurfer.js@7/dist/plugins/timeline.min.js"></script>

I try to initialize it:

wavesurfer.registerPlugin(TimelinePlugin.create())

But get the error. 'Uncaught ReferenceError: TimelinePlugin is not defined'

Do I have to use TS instead of JS?

**Comment by katspaugh** [comment, marked as answer]
When you use a `min.js` file, it's `WaveSurfer.Timeline`, not `TimelinePlugin`.

From the readme:
<img width="878" alt="Screenshot 2023-10-18 at 12 03 40" src="https://github.com/katspaugh/wavesurfer.js/assets/381895/f1bbfdd2-3988-4c0e-880b-c9d68f870e22">

**Reply by ksaugmann** [reply]
I see, thanks for the quick reply!
